### PR TITLE
add issue template

### PR DESCRIPTION
**Pull request by gaoning777**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1492)

### CODE REVIEW

**Comment by gaoning777** [comment]
@IronPan Could you take a look when you have time?

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Bug Report
+about: Report a bug encountered with KFP
```
Comment (IronPan) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]: KFP -> Kubeflow Pipelines

**Comment by IronPan** [comment]
please address the comment then unhold.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md (hunk 1 above)
Comment (gaoning777) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md]: Done.

**Comment by gaoning777** [comment]
@IronPan Looks like I cannot lgtm my own PR.

**Pull request merged**